rs-e2e/README.md

```diff
@@ -13,14 +13,44 @@ Information on how to set up the sample files evaluated by the tests can be foun
 
 ## Running the tests
 
-- Automatically - these are scheduled to run every weekday
-- Manually via Github
-  - Run the [automated-staging-test-submit](/.github/workflows/automated-staging-test-submit.yml) action
-  - Wait for RS and TI to finish processing files
-  - Run the [automated-staging-test-run](/.github/workflows/automated-staging-test-run.yml) action
-- Locally
-  - Set the `AZURE_STORAGE_CONNECTION_STRING` environment variable to the [value in Keybase](keybase://team/cdc_ti/service_keys/TI/staging/azure-storage-connection-string-for-automated-rs-e2e-tests.txt)
-  - Run the tests with `./gradlew rs-e2e:clean rs-e2e:automatedTest`
+### Automatically
+
+There are two scheduled tasks that run every weekday around midnight EST:
+
+- [Automated Staging Test - Submit Messages](https://github.com/CDCgov/trusted-intermediary/actions/workflows/automated-staging-test-submit.yml) submits the messages in `/examples/Test/Automated`
+- [Automated Staging Test - Run integration tests](https://github.com/CDCgov/trusted-intermediary/actions/workflows/automated-staging-test-run.yml) triggers a couple of hours later and runs the Automated Tests on the input files in `/examples/Test/Automated` and the output files in the Azure storage container.
+
+### Manually from your local machine
+
+When running locally, we usually run the tests from either the command line using gradle, or from IntelliJ. Please note that even though you are running the test from your local machine, the test will need to connect to the Azure container to pull the output files to apply the assertions on. For this reason, you will need to set the `AZURE_STORAGE_CONNECTION_STRING` environment variable to authenticate the connection to Azure. You can find the value for `AZURE_STORAGE_CONNECTION_STRING` in Keybase (keybase://team/cdc_ti/service_keys/TI/staging/azure-storage-connection-string-for-automated-rs-e2e-tests.txt).
+
+From the command line:
+
+1. Set the `AZURE_STORAGE_CONNECTION_STRING` environment variable in your shell
```

Comment:
The only thing this is missing to me is that nothing says what this env var should be set to anymore.

Reply:
It is in the paragraph above

Reply:
Oh, I should've checked the full file and not the div. Well the PR is all good then! 👍